js/base/Exchange.js

```diff
@@ -2222,4 +2224,42 @@ module.exports = class Exchange {
         }
 
     }
+
+    /**
+     * @param {string} type: Order type
+     * @param {string} timeInForce
+     * @param {boolean} exchangeSpecificOption: True if the exchange specific post only setting is set
+     * @param {dict} params: Exchange specific params
+     * @returns {boolean}: true if a post only order, false otherwise
+     */
+    isPostOnly (type, timeInForce, exchangeSpecificOption, params = {}) {
+        let postOnly = this.safeValue2 (params, 'postOnly', 'post_only', false);
+        params = this.omit (params, ['post_only', 'postOnly']);
+        const timeInForceUpper = timeInForce.toUpperCase ();
+        const typeLower = type.toLowerCase ();
+        const ioc = timeInForceUpper === 'IOC';
+        const timeInForcePostOnly = timeInForceUpper === 'PO';
```

Comment:
I think the only allowable `timeInForce` for postOnly orders should be `PO`

Reply:
GTC is valid, there's also some exchanges that specify GTX, postOnly ...

Reply:
I added FOK, but GTC should stay